src/atom.ts

```diff
@@ -21,10 +21,16 @@ export function atom<Value, Update = unknown>(
   write?: (update: Update) => void,
 ) {
   if (read instanceof Function) {
-    return derivedAtom(read, write)
+    if (write) {
+      return customAtom(read, write)
+    }
+    return derivedAtom(read)
   }
+  return basicAtom(read)
+}
 
-  const subject = cachedSubject(read)
+const basicAtom = <Value>(value: Value): Atom<Value> => {
```

Comment:
Not neccesarily wrong, but I get the feeling taht basic kinda suggests that it is less-than-other atoms?

In a sense, it is the _base_ on which all other atoms build, so basic *is* a good term.

But I just wanna throw out some other suggestions:

- baseAtom
- primitiveAtom
- foundationalAtom
- cruxAtom